app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Configuration for the esbuild bundler and build/watch commands.
+ */
+
+import fs from 'node:fs'
+import path, { dirname } from 'node:path'
+import child_process from 'node:child_process'
+import { fileURLToPath } from 'node:url'
+
+import esbuild from 'esbuild'
+import plugin_yaml from 'esbuild-plugin-yaml'
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill'
+import aliasPlugin from 'esbuild-plugin-alias'
+// @ts-ignore
+import timePlugin from 'esbuild-plugin-time'
+// @ts-ignore
+import * as copy_plugin from 'enso-copy-plugin'
+
+import { require_env } from '../../utils.js'
+import * as BUILD_INFO from '../../build.json' assert { type: 'json' }
+
+// =============================
+// === Environment variables ===
+// =============================
+
+export const thisPath = path.resolve(dirname(fileURLToPath(import.meta.url)))
+/** Path where bundled files are output. */
+export const output_path = path.resolve(require_env('ENSO_BUILD_GUI'), 'assets')
+export const wasm_path = require_env('ENSO_BUILD_GUI_WASM')
+export const js_glue_path = require_env('ENSO_BUILD_GUI_JS_GLUE')
+export const assets_path = require_env('ENSO_BUILD_GUI_ASSETS')
+
+// ===================
+// === Git process ===
+// ===================
+
+/**
+ * Get output of a git command.
+ * @param command Command line following the `git` program.
+ * @returns Output of the command.
+ */
+function git(command: string): string {
+    // TODO [mwu] Eventually this should be removed, data should be provided by the build script through `BUILD_INFO`.
```

Comment:
no link to pivotal and no explanation of what happens if this todo is not done.

Reply:
Nothing happens, I just don't like it. No point in adding to pivotal (no one would ever schedule this), this is more of a note to developer that if this file is refactored, this particular thing could be improved.

I've expanded the comment.